packages/components/src/public/QueryModel/TabbedGridPanel.tsx

```diff
@@ -114,6 +126,42 @@ export const TabbedGridPanel: FC<TabbedGridPanelProps & InjectedQueryModels> = m
         },
         [onTabSelect]
     );
+
+    const exportTabs = useCallback(async (selectedTabs: string[]|Set<string>):Promise<void> => {
+        try {
+            // set exporting blocker
+            setCanExport(false);
+            let models = [];
+            selectedTabs.forEach(selected => models.push(queryModels[selected]?.schemaQuery));
+            const filename = exportFilename ?? 'Data';
+            await exportTabsXlsx(filename, models);
+            onExport?.[EXPORT_TYPES.EXCEL]?.();
+            createNotification({message: 'Successfully exported tabs to file.', alertClass: 'success'});
+        } catch (e) {
+            //Set export error
+            createNotification({message: 'Export failed: ' + e.getErrorMessage(), alertClass: 'danger'});
+        } finally {
+            // unset exporting blocker
+            setShowExportModal(false);
+            setCanExport(true);
+        }
+    }, [exportFilename, canExport, queryModels]);
+
+    const exportHandler = useCallback(async () => {
```

Comment:
I was a bit confused at first thinking that all export types would call this callback, but it is just excel (based on the definition of `exportHandlers` at line 159. I might suggest renaming this to `excelExportHandler` just to self document.